src/main/groovy/com/palantir/gradle/javadist/tasks/DistTarTask.groovy

```diff
@@ -43,7 +43,10 @@ class DistTarTask extends Tar {
                 }
             }
 
-            new File(project.projectDir, "var/data/tmp").mkdirs()
+            new File(project.buildDir, "distributions/var/data/tmp").mkdirs()
+            from ("${project.buildDir}/distributions/var/data") {
```

Comment:
Can clarify in the readme, but the exclusion is copying `${projectDir}/var/data`, which this still respects.